lib/ladybug/energy_model/model.rb

```diff
@@ -39,115 +58,186 @@ module Ladybug
+            construction_object = OpaqueConstructionAbridged.new(construction)
+          when 'WindowConstructionAbridged'
+            construction_object = WindowConstructionAbridged.new(construction)
+          when 'ShadeConstruction'
+            construction_object = ShadeConstruction.new(construction)
           else
-            @errors << "Unknown face_type '#{face_type}' for face '#{name}', surface not created"
-            next
+            raise "Unknown construction type #{construction_type}."
           end
-          
-          vertices = OpenStudio::Point3dVector.new
-          if @face_by_face
-            # vertices in face
-            face[:vertices].each do |v|
-              vertices << OpenStudio::Point3d.new(v[0], v[1], v[2])
-            end 
-          else
-            # vertices in separate list
-            face[:vertices].each do |vi|
-              v = @model[:vertices][vi]
-              vertices << OpenStudio::Point3d.new(v[0], v[1], v[2])
-            end 
+          construction_object.to_openstudio(@openstudio_model)
+        end
+      end
+
+      def create_construction_set
+        if @hash[:properties][:energy][:construction_sets]
+          @hash[:properties][:energy][:construction_sets].each do |construction_set|
+          construction_set_object = ConstructionSetAbridged.new(construction_set)
+          construction_set_object.to_openstudio(@openstudio_model)
           end
-               
-          surface = OpenStudio::Model::Surface.new(vertices, osm)
-          surface.setName(name)
-          surface.setSpace(space)
-          surface.setSurfaceType(surface_type) if surface_type
-          if air_wall
-            # DLM: todo
+        end
+      end
+
+      def create_global_construction_set
+        openstudio_construction = nil
+        if @hash[:properties][:energy][:global_construction_set]
+          construction_name = @hash[:properties][:energy][:global_construction_set]
+          construction = @openstudio_model.getDefaultConstructionSetByName(construction_name)
+          unless construction.empty?
+            openstudio_construction = construction.get
           end
+          @openstudio_model.getBuilding.setDefaultConstructionSet(openstudio_construction)
         end
       end
-      
-      def create_apertures(osm)
+
+      def create_rooms
+        if @hash[:rooms] 
+          @hash[:rooms].each do |room|
+          room_object = Room.new(room)
+          room_object.to_openstudio(@openstudio_model)
+          end
+        end
+      end
+
+      def create_orphaned_shades
+        if @hash[:orphaned_shades]
+          @hash[:orphaned_shades].each do |shade|
+          shade_object = Shade.new(shade)
+          shade_object.to_openstudio(@openstudio_model)
+          end
+        end
+      end
+#runlog
+      def create_orphaned_faces
+        if @hash[:orphaned_faces]
+          raise "Face is not translatable to OpenStudio object."
```

Comment:
Thanks for including these warnings in here.  For clarity, I think they should read:
`Orphaned Faces are not translatable to OpenStudio`
The same goes for the other warnings here.